sakai-evaluation-api/src/java/org/sakaiproject/evaluation/model/EvalEvaluation.java

```diff
@@ -422,7 +432,26 @@ public EvalEvaluation(String type, String owner, String title, String instructio
         this(type, owner, title, instructions, startDate, dueDate, stopDate, viewDate, studentViewResults, studentsDate, instructorViewResults, instructorViewAllResults, instructorsDate, state,
                 resultsSharing, instructorOpt, reminderDays, reminderFromEmail, termId, availableEmailTemplate, reminderEmailTemplate, template,
                 responses, blankResponsesAllowed, modifyResponsesAllowed, unregisteredAllowed, Boolean.FALSE ,locked, authControl,
-                evalCategory, selectionSettings, Boolean.TRUE);
+                evalCategory, selectionSettings, Boolean.TRUE, null);
```

Comment:
Why pass null here instead of a default value (false I would imagine in this case)?